workers/README.md

```diff
@@ -0,0 +1,35 @@
+# Worker
+
+## Installation
+Ensure that pip and python is installed
+
+`````
+install pipenv for dependencies management
+pip install --user pipenv
+make sure the path is added to the system
+`````
+install Requests library for HTTP request
+pipenv install requests
+list of dependencies will be listed on Pipfile
+
+
+## Authentication for Twitter
+Create an account folloed by an app
+https://developer.twitter.com/en/account/get-started
+`````
+Get your application tokens at "Keys and Access Tokens"
+consumer key, consumer secret key, access token, access token secret 
+`````
+Use get_bearer_token.py to get Bearer Token 
+Bearer Token is required to use the Request lib for Twitter API
+`````
+
+`````
```

Comment:
Why are there so many empty code blocks in your markdowns? Please change them across the whole repo.
Refer to :
- https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax
- https://guides.github.com/features/mastering-markdown/